src/org/opensolaris/opengrok/configuration/messages/Message.java

```diff
@@ -100,6 +103,14 @@ public static Message createMessage(String type) {
         return null;
     }
 
+    public Configuration getConfiguration() {
```

Comment:
I'd rather introduce a field called `file` containing the content. Configuration has no meaning for any other type of message however we can use appending a file content in the future somehow.

Actually you can change it so it uses the `text` field for a file content and use `tags` for signalizing the reindex without the need to change the abstract type.